### ISSUE TITLE
keybase: 0.8.25 -> 1.0.15

### ISSUE THREAD

**Issue by choochootrain** [NixOS/nixpkgs#14773]
## Issue description

keybase nodejs client is deprecated, new client is here: https://github.com/keybase/client/releases
### Steps to reproduce

```
$ nix-env -i keybase
$ keybase --version
```
## Technical details

```
$ nixos-version
16.03.git.e409886 (Emu)

$ nix-env --version)
nix-env (Nix) 1.11.2

$ nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion
"16.03.git.e409886"
```

**Comment by choochootrain** [comment]
i'm a week into nixos but i would be more than happy to take this on if i could get some pointers.

it seems like updating the derivation should be enough, but is there an idiomatic way to migrate a package's metadata to a new version? i tried poking around some of the postgres stuff but i'm a little lost

**Comment by OJFord** [comment]
Looks like this can be closed since the Keybase package is @ 1.0.18 now.

**Comment by fpletz** [comment]
@OJFord Thanks for the notice!
